libs/dh/admin/data-access-api/src/lib/dh-admin-user-roles-management-data-access-api.store.ts

```diff
@@ -197,19 +111,17 @@ export class DhAdminUserRolesManagementDataAccessApiStore
   };
 
   ngrxOnStoreInit(): void {
-    this.rolesOptions$ = this._transloco
-      .selectTranslateObject('marketParticipant.marketRoles')
-      .pipe(
-        combineLatestWith(this.select((state) => state.roles)),
-        filter(([, roles]) => roles.length > 0),
-        // eslint-disable-next-line @ngrx/avoid-mapping-component-store-selectors
-        map(([keys, roles]) =>
-          roles.map((role: MarketParticipantUserRoleDto) => ({
-            displayValue: `${role.name} (${keys[role.eicFunction]})`,
-            value: role.id,
-          }))
-        )
-      );
+    this.rolesOptions$ = this.transloco.selectTranslateObject('marketParticipant.marketRoles').pipe(
+      combineLatestWith(this.select((state) => state.roles)),
+      filter(([, roles]) => roles.length > 0),
+      // eslint-disable-next-line @ngrx/avoid-mapping-component-store-selectors
+      map(([keys, roles]) =>
+        roles.map((role: UserRoleItem) => ({
+          displayValue: `${role.name} (${keys[role.eicFunction]})`,
```

Comment:
Suggestion: Lets make a displayValue on the UserRoleItem in BFF, like we have with actor and gridarea

Reply:
This will not work with translations. The `eicFunction` value is translated here.
I can see that the same issue appears in the "Market participant" dropdown on the Users page.